packages/eslint-plugin/src/configs/react-legacy.js

```diff
@@ -10,5 +10,6 @@ module.exports = {
   rules: {
     ...configHelpers.getNamingConventionRule(true /* prefixWithI */),
     'jsdoc/check-tag-names': 'off',
+    '@griffel/no-shorthands': 'off',
```

Comment:
v8 has own `makeStyles`, new rule collides with it.

Reply:
not related to this PR, but what are partners supposed to do in their repo when they have both v8 and v9?

Reply:
Good catch, I am not sure that `makeStyles` in v8 is a recommended approach:

https://github.com/microsoft/fluentui/blob/0915feeaa4278c32a7818b67b159ed207d1031fc/packages/react/src/utilities/ThemeProvider/makeStyles.ts#L92-L93

So for that usages that probably just ignore files for that rule.

Reply:
> not related to this PR, but what are partners supposed to do in their repo when they have both v8 and v9?

I'd say it should be up to them to enable linting rules that they need. our monorepo eslint plugin/config is not in the best shape and we don't follow proper semver for it, so I would not recommend to any partner to use our configurations